src/Elmish.WPF.Tests/UtilsTests.fs

```diff
@@ -95,3 +95,70 @@ module elmEq =
         let t2 = { t2 with t = t1.t }
         test <@ elmEq t1 t2 = (t1.i = t2.i && t1.s = t2.s) @>
       }
+
+
+module ValueOption =
+
+  module toNullable =
+
+    type ToNullableError =
+      | NullReference
+      | OtherExn of exn
+
+    let tryToNullable<'a> (a: 'a voption) =
+      try
+        ValueOption.toNullable a |> Ok
+      with
+      | :? System.NullReferenceException -> NullReference |> Error
+      | ex -> ex |> OtherExn |> Error
+
+    let testNonNull (ga: Gen<'a>) =
+      Property.check <| property {
+        let! expected = ga
+        test <@ expected = (expected |> ValueSome |> ValueOption.toNullable) @>
+      }
+
+    [<Fact>]
+    let ``toNullable returns ValueSome contents when given ValueSome`` () =
+      testNonNull GenX.auto<obj>
+      testNonNull GenX.auto<string>
+      testNonNull GenX.auto<int>
```

Comment:
```suggestion
```

Reply:
We need to test non-nullable types as the name of the test implies